meshtastic/interfaces/ble/management_service.py

```
@@ -360,7 +386,9 @@ def _execute_with_client(
             if temporary_client is not None:
                 try:
                     iface._client_manager_safe_close_client(temporary_client)
-                except Exception:  # noqa: BLE001 - best-effort cleanup must not mask command outcome
+                except (
+                    Exception
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)